rbc/tests/test_omnisci_column_arguments.py

```diff
@@ -0,0 +1,111 @@
+import pytest
+from rbc.tests import omnisci_fixture
+
+
+@pytest.fixture(scope='module')
+def omnisci():
+
+    for o in omnisci_fixture(globals(), minimal_version=(5, 6)):
+        define(o)
+        yield o
+
+
+def define(omnisci):
+
+    @omnisci('int32(Column<int64>, RowMultiplier, OutputColumn<int64>)')
+    def text_rbc_copy_c_rowmul(x, m, y):
+        for i in range(len(x)):
+            # TODO: implement is_null support
```

Comment:
This "TODO" refers to PR #230?

Reply:
yes, right now the copy table functions are used for testing the rbc issue #187 but after the null support is in, these tests can be used for testing is_null support as well.